ZCRMiOS/Classes/ZCRMOrganisation.swift

```diff
@@ -399,6 +399,20 @@ public class ZCRMOrganisation : ZCRMEntity
             completion( role, response, error )
         }
     }
+    
+    public func searchUserByCriteria( criteria : String, completion : @escaping( [ ZCRMUser ]?, BulkAPIResponse?, Error? ) -> () )
```

Comment:
have method name as "searchUsers" and label for first argument as "withCriteria"